articles/azure-app-configuration/howto-import-export-data.md

```diff
@@ -455,10 +455,10 @@ From the Azure portal, follow these steps:
 
     | Parameter          | Description                                                                                                                                                                                                                            | Example                   |
     |--------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------|
-    | Prefix             | Optional. This prefix will be trimmed from the imported keys. A key prefix is the beginning part of a key. Prefixes can be used to manage groups of keys in a configuration store. Prefix will be ignored for feature flags.           | *TestApp:*                 |
+    | Prefix             | Optional. This prefix will be trimmed from the exported keys. A key prefix is the beginning part of a key. Prefixes can be used to manage groups of keys in a configuration store. Prefix will be ignored for feature flags.           | *TestApp:*                 |
```

Comment:
The "prefix" item is in the UI there. It's not something you can hide. I know it's an optional parameter for CLI, but the "optional" doesn't make sense for UI in the portal.

Reply:
I agree that this cannot be hidden for the portal, however, in my opinion, "Optional" is a way of saying this field can be left blank. What do you think?

Reply:
Empty is also a value provided by the user, but sure, I can see your point.

You could call the "time" field as optional. Then please explain what's the default value.